config/routes.rb

```diff
@@ -1,21 +1,22 @@
-Rails.application.routes.draw do
+# frozen_string_literal: true
 
+Rails.application.routes.draw do
   root to: redirect("/#{I18n.default_locale}"), as: :redirected_root
-  scope "(:locale)", locale: /#{I18n.available_locales.join("|")}/ do
-
-    devise_for :users 
+  scope '(:locale)', locale: /#{I18n.available_locales.join("|")}/ do
+    devise_for :users
 
     resources :categories
     root 'static_pages#home'
     get '/about', to: 'static_pages#about'
-  
+
     resources :lessons do
       resources :comments
     end
 
     resources :users do
       resources :comments
     end
-  end
 
+    resources :admins, controller: 'users'
```

Comment:
Why do we have resources :admins? It will be nice to have scope or namespace :admin and url will looks like admin/users
Controller will be Admin::UsersController.